code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -159,8 +159,6 @@
 	toner = tonermax
 	diag_hud_set_borgcell()
 
-	verbs += /mob/living/proc/lay_down //CITADEL EDIT borgs have rest verb now for snowflake reasons
```

Comment:
Readd this, I need to pet those good robobois

Reply:
It doesn't work